piet-web/README.md

```diff
@@ -1,11 +1,11 @@
 # Running the examples
 
-Generally follow the directions at [stdweb].
+Ensure both cargo and [npm] are installed.
 
-`$ cargo install -f cargo-web`
+`$ cargo install -f wasm-bindgen`
```

Comment:
I get this:

```
$ cargo install -f wasm-bindgen
    Updating crates.io index
  Installing wasm-bindgen v0.2.30
error: specified package has no binaries
```

I can diagnose this, but I have on my "hapless dev" hat. This is on a Windows 10 box in a git bash shell, and node (10.15.0) and npm are installed.

Reply:
Whoops! It's actually `wasm-bindgen-cli`. Once it's installed it's just called `wasm-bindgen`. I will update this.